### PR TITLE
alsa_in.c alsa_out.c

### PR DESCRIPTION
**Pull request by fjapinteric**
Added -S option and supporting code to permit attaching to a named jack2 server.

### CODE REVIEW

**Comment by falkTX** [comment]
there's a minor thing about spaces vs tabs, but the rest seems fine to me.
I'll let others comment as well.

**Comment by kmatheussen** [comment]
Looks fine to me. Also looked at the source code of jack to make sure it ignores an extra NULL argument.

File: example-clients/alsa_in.c [hunk 1]
```diff
@@ -683,6 +686,10 @@ int main (int argc, char *argv[]) {
 	    case 's':
 		smooth_size = atoi(optarg);
 		break;
+		case 'S':
```
Comment (falkTX) [inline example-clients/alsa_in.c]: indentation on these lines do not match the rest of the file.

File: example-clients/alsa_out.c [hunk 2]
```diff
@@ -686,6 +688,10 @@ int main (int argc, char *argv[]) {
 	    case 's':
 		smooth_size = atoi(optarg);
 		break;
+		case 'S':
```
Comment (falkTX) [inline example-clients/alsa_out.c]: same here, indentation is off.

**Comment by 7890** [comment]
I will merge this PR and fix the indentation manually.

**Pull request merged**